docs/resources/samples/add-excel-comments.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'Add comments in Excel'
+description: 'Learn how to use Office Scripts to add comments in a worksheet.'
+ms.date: 03/29/2021
+localization_priority: Normal
+---
+
+# Add comments in Excel
+
+This sample shows how to add comments to a cell including [@mentioning](https://support.microsoft.com/office/90701709-5dc1-41c7-aa48-b01d4a46e8c7) a colleague.
+
+## Example scenario
+
+* The team lead maintains the shift schedule. The team lead assigns an employee ID to the shift record.
+* The team lead wishes to notify the employee. By adding a comment that @mentions the employee, the employee is emailed with a custom message from the sheet.
+* The employee can then address and respond to the comment.
```

Comment:
Should we mention this isn't in the script?
```suggestion
* The employee can view the workbook and respond to the comment at their convenience.
```

Reply:
I don't think so. The scenario seems to discuss a usage flow then the solution section discuss what the script actually does.